pages/dashboard.tsx

```diff
@@ -0,0 +1,26 @@
+import NumberDisplay from '../components/NumberDisplay';
+import Warnings from '../components/Warnings';
+import Graph from '../components/Graph';
+
+const Dashboard = () => {
+  return (
+    <div className="bg-black w-screen h-screen px-4">
+      <h1 className="mt-24 text-ocf-gray text-3xl font-bold">Dashboard</h1>
+      <div className="flex flex-row w-full justify-start">
+        <Warnings />
+      </div>
+      <div className="flex flex-row w-full justify-start">
+        <NumberDisplay title="Today's Expected Output" value="2700kW" />
+      </div>
+      <div className="flex flex-row w-full justify-center space-x-4">
+        <NumberDisplay title="Current Output" value="2200kW" />
+        <NumberDisplay title="Current Capacity" value="80%" />
+      </div>
+      <div className="flex flex-row w-full justify-start">
+        <Graph />
+      </div>
+    </div>
+  );
+};
+
+export default Dashboard;
```

Comment:
Since the dashboard should be visible to authenticated users only, let's add 
``export const getServerSideProps = withPageAuthRequired();``
to the end of this file! That should get the page to redirect unauthenticated users to log back in before they can access this page. 
```suggestion
export default Dashboard;
export const getServerSideProps = withPageAuthRequired();
```